packages/split-button/src/SplitButton.ts

```diff
@@ -115,6 +115,14 @@ export class SplitButton extends SizedMixin(PickerBase) {
                 this.selects = 'single';
             }
         }
+        if (window.__swc.DEBUG) {
+            window.__swc.warn(
+                this,
+                `<${this.localName}> is deprecated. Use a [button group](https://opensource.adobe.com/spectrum-web-components/components/button-group/) to show any additional actions related to the most critical action.`,
+                'https://opensource.adobe.com/spectrum-web-components/components/button-group/',
```

Comment:
This should likely point to a “Deprecated” section that we add to the Split Button readme to ensure the same message is received by devs who do not leverage DEV Mode.

Reply:
Yes that's the final motive. I have added the change to be added here in the description

Reply:
<img width="1114" alt="Screenshot 2024-03-04 at 4 16 29 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/add1852a-3c17-421d-8bce-58059c0488f4">